### PR TITLE
adjusts footer to move back to bottom

### PR DESCRIPTION
**Pull request by weyermannx**
..when new user closes the new user message

https://ama-digital.myjetbrains.com/youtrack/issue/OAUTH-1089

### CODE REVIEW

File: lib/ama_layout/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module AmaLayout
-  VERSION = "2.0.6"
+  VERSION = "2.0.7"
```
Comment (darkodosenovic) [inline lib/ama_layout/version.rb]: Adjust version. Should be over 2.1.x I assume.

**Comment by darkodosenovic** [comment]
:poop: 

Merge master

**Comment by weyermannx** [comment]
:wave:

**Comment by darkodosenovic** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**